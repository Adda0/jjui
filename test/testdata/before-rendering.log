◆  sm;smwykmvq;main;false;true;false;false;false;some@email.com;1 day ago;fix: clean up change id from graph characters
~  (elided revisions)
│ ○  up;upzzyuvs;.;false;false;false;false;false;some@email.com;2 months ago;feat: add key manager for better key handling
├─╯
◆  sw;swkztzll;.;false;true;false;false;false;some@email.com;2 months ago;refactor: move diff view out of revisions
~  (elided revisions)
│ ○  vx;vxvtkvrz;.;false;false;false;false;false;some@email.com;2 months ago;wip: refactor: use different renderers in revisions component depending on mode
├─╯
◆  vn;vnzyvqkx;.;false;true;false;false;false;some@email.com;2 months ago;refactor: move help rendering out of revisions
~  (elided revisions)
│ ○  t;tnwwkmry;.;false;false;false;false;false;some@email.com;2 months ago;refactor: show operation windows as detail view
├─╯
│ ○  r;rnymlszl;.;false;false;false;false;false;some@email.com;2 months ago;wip: try viewport to draw some parts of the revision graph
├─╯
◆  uo;uolvlvsq;.;false;true;false;false;false;some@email.com;2 months ago;refactor: extract styles and renderer into their own files
│
~