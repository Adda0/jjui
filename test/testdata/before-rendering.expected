◆  smwykmvq
│  fix: clean up change id from graph characters
~  (elided revisions)
│    <here>
│ ○  upzzyuvs
├─╯  feat: add key manager for better key handling
◆  swkztzll
│  refactor: move diff view out of revisions
~  (elided revisions)
│ ○  vxvtkvrz
├─╯  wip: refactor: use different renderers in revisions component depending on mode
◆  vnzyvqkx
│  refactor: move help rendering out of revisions
~  (elided revisions)
│ ○  tnwwkmry
├─╯  refactor: show operation windows as detail view
│ ○  rnymlszl
├─╯  wip: try viewport to draw some parts of the revision graph
◆  uolvlvsq
│  refactor: extract styles and renderer into their own files
~  (elided revisions)